haystack/preview/document_stores/memory/document_store.py

```diff
@@ -250,7 +250,7 @@ def bm25_retrieval(
         # get scores for the query against the corpus
         docs_scores = bm25_scorer.get_scores(tokenized_query)
         if scale_score:
-            docs_scores = [float(expit(np.asarray(score / SCALING_FACTOR))) for score in docs_scores]
```

Comment:
Is this change intentional?

Reply:
> I had to fix a weird call to expit that was exposed by mypy

Yes, that call is wrong, like many others (that I won't fix in this PR).